docs/EnvironmentVariables.md

```diff
@@ -21,3 +21,14 @@
 * `ZM_BACKEND_RPCAPI_LOGJSON`: Setting it to any thruthy value (non-empty
   string or non-zero number) will configure the logger to log in JSON format,
   undefined by default.
+
+## Variables specific to testing
+
+* `TARGET`: Set the database to use. If undefined, will assume SQLite.
+  Accepted values are:
+  * `SQLite`
+  * `PostgreSQL`
+  * `MySQL`
+
+* `ZONEMASTER_RECORD`: If set, the data from the test is recorded to a file.
+  Otherwise the data is loaded from a file.
```

Comment:
Is "set" correct here? Shouldn't it be "if true"? If set to 0 or empty string will give the same affect as unset, won't it?

Reply:
fixed